qstash/workflows/getstarted.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: "Getting Started"
+---
+
+**Workflows** is a Durable Execution Engine that is composed of **Qstash** Server and the SDK.
+
+- It allows to break **free of time limitations** of serverless platforms.
+- It enables **recovery from partial failures**, with the ability to continue a workflow run from an intermediate step.
+- It designed to seamlessly integrate with **your existing infrastructure**. The application code will not be uploaded to QStash.
+- It provides **all the power of QStash** while abstracting away the complexities, allowing you to focus on implementing the **essential parts**.
+
+Here is an example how an an endpoint might use our SDK:
+
+```javascript 
+import { serve } from "@upstash/qstash/workflow/nextjs";
+import { sendEmail, UserQuestionRequest } from "./utils"
+
+export const POST = serve<{userRequest: UserQuestionRequest}>({async context => {
+    await context.sleep("sleepDays" , 3d)
```

Comment:
At first glance, the logic of immediately sleeping for 3 days doesn’t make sense to me. A more logical flow might be:

1.	Save the user to the database.
2.	Sleep for the specified duration.
3.	Make the external call and send the email.

WDYT?

Also, for the sleep durations, I noticed you’re using invalid inputs like `3d` or utility functions. Would it be clearer to use seconds directly, like `3 * 24 * 60 * 60`? Since people might copy-paste these code snippets as templates, it would be better to make them directly usable. 

```suggestion
    await context.run("saveUser", async() => {
        await saveUser(userRequest)
    })
    
    await context.sleep("sleepDays" , 3d)
```